app/models/case/sar/offender.rb

```diff
@@ -239,6 +241,18 @@ def validate_third_party_address
     errors[:third_party_relationship].any?
   end
 
+  def validate_third_party_email
+    if third_party
+      return if third_party_email.blank?
+      unless third_party_email =~ /\A([^@,]+)@([^@,]+)\z/ # regex disallows commas and additional @s
```

Comment:
There's a email validation no case which might be better to copy - https://github.com/ministryofjustice/correspondence_tool_staff/blob/main/app/models/case/base.rb#L967

Reply:
This regex breaks the third_party_email validation and browser hangs

Reply:
In this commit - https://github.com/ministryofjustice/correspondence_tool_staff/pull/2318/commits/246099428db673eac0996a55b4eca28671fd0155 - you change the attribute to `email` instead of `third_party_email`